account_product_fiscal_classification/models/account_product_fiscal_classification_model.py

```diff
@@ -33,13 +15,15 @@ class AccountProductFiscalClassificationModel(models.AbstractModel):
 
     # Field Section
     code = fields.Char()
-
     name = fields.Char(
-        size=_MAX_LENGTH_NAME, required=True, select=True, translate=True)
-
+        size=_MAX_LENGTH_NAME,
+        required=True,
+        index=True,
+        translate=True
+    )
```

Comment:
don't understand such changes ? is it an OCA convention to set an argument by line ?

(and other in other python file).

(not a blocking point, just a remark)